Hai Reveal conversations: reduce text walls in hai reveal 3 early (#7567)
endless-sky/endless-sky: data/hai/hai reveal 3 early.txt

			`	"I am very glad to see you returned, Captain <last>. Sayari said you were away on an important errand, but I've been hoping for a day now that you would reappear. I think you might be the solution to the current impasse."`
			`	"You see," says Sayari, "while you've been away it's been decided that a 'tourist trap type theme park destination' is a critical part of the cover story." Despite her Hai inflections, you can still hear the air quotes around that phrase in the tone of her voice. "So far it seems the movie cover has been circulating quite successfully, so we're leaning into it.`
			`	"This means we need a station, on your side of the wormhole in Ultima Thule. It also means we need it fast, much faster than your people could ever hope to assemble it. The Hai can build it much quicker than humanity can; we actually considered the possibility of needing to raise a station quickly a while back. The alien look that the station will have if we build it has been considered just another favorable aspect of the movie and it provides a long-term commercial justification for the whole idea; the, uh, 'tourist trap'.`
			choice

> 				`	"I should oversee the construction, given the current political situation."`

Reviewer: Leads into the response better.
```suggestion
				`	"It would help if I oversaw the construction, wouldn't it?"`
```

					goto oversee
				`	"Good idea. Let me know when the station is complete."`

> 			`	"It's not so simple." Sayari continues, "In the light of the recent civil war your species has had, some of the Elders are less than keen on allowing a partisan observer to see the inner workings of a station in its earliest phases, especially when we haven't actually built one in so, so long."`

Reviewer 2: ```suggestion
			`	"It's not so simple," Sayari responds. "In the light of the recent civil war your species has had, some of the Elders are less than keen on allowing a partisan observer to see the inner workings of a station in its earliest phases, especially when we haven't actually built one in so, so long."`
```

			`	"Anyway," she continues, "thanks to a little Freedom of Information request to the Republic, I have been looking at things, and I think I have the solution."`
			choice
				`	"Is it me?"`

> 			`	Sayari winks at you, "Catch on quick, don't you? The actions you're recorded as having taken show a certain conscience for the good of your species as a whole, despite your affiliations. I have convinced the other diplomats and our Elders that you might be a viable solution to this impasse. The human diplomats have already agreed, but the Elders wish to speak with you first."`

Reviewer 2: ```suggestion
			`	Sayari winks at you. "Catch on quick, don't you? The actions you're recorded as having taken show a certain conscience for the good of your species as a whole, despite your affiliations. I have convinced the other diplomats and our Elders that you might be a viable solution to this impasse. The human diplomats have already agreed, but the Elders wish to speak with you first."`
```

				goto elders
			label oversee
			`	"You're learning to navigating the maze of galactic politics quickly, Captain <last>," says Sayari. "We are at an impasse. The Elders are less than keen on allowing a partisan observer to see the construction."`

> 			`	Xilin continues her thought, "The Republic won't let the Hai build a station on our side of the wormhole without a trusted observer monitoring construction.`

Reviewer 2: ```suggestion
			`	Xilin continues her thought. "The Republic won't let the Hai build a station on our side of the wormhole without a trusted observer monitoring construction.`
```

> 			`	"Both sides trust you though", he says, as Sayari nods.`

Reviewer 2: ```suggestion
			`	"Both sides trust you, though," he says, as Sayari nods.`
```

			`	Gingerly, she takes the data stick and secretes it away on her person. Her expression of distaste looks like one she might give to some snake that could bite her at any moment.`
			`	"I will have a thorough look later," is all she says.`
			`	No sooner has she finished than there is a knock at the door, and an aide looks in. "The Council will see you now."`
			choice

> 				`	Walk to the Elder Council chambers.`

Reviewer 2: ```suggestion
				`	(Walk into the Council's chambers.)`
```

				`	"What's the problem?"`
					goto problem
				`	"The movie production is real?"`

> 			`	She looks blinks at you as if caught slightly off-guard. "Yes, didn't they tell you? In order to keep up the cover they actually have to start making a movie, or at least believable clips from one. The director they found, an eccentric chap called David Cohen, seems to have got quite enthusiastic now he's not busy bemoaning the 'big reveal' of his 'secret project.' There's a Mr Turner who's now the cover story's guy bankrolling the thing.`

Reviewer 2: I'm not sure whether this part should be changed, what you've done looks correct to me but I've been told to leave them as it is.

Author: I'll leave my change in, and see who comments about it.

Reviewer 3: "She looks blinks" doesn't sound correct.

Author: > "She looks blinks" doesn't sound correct.

Indeed. Thanks for noticing that. I'll fix it in my next round of textual corrections in 5-10 hours. Please let me know if anything else is wrong.

Author: It appears an earlier commit already changed that to "looks and blinks" but github didn't update this comment.

```
`       She looks and blinks at you as if caught slightly off-guard. "Yes, didn't they tell you?
```

> 			`	She looks blinks at you as if caught slightly off-guard. "Yes, didn't they tell you? In order to keep up the cover they actually have to start making a movie, or at least believable clips from one. The director they found, an eccentric chap called David Cohen, seems to have got quite enthusiastic now he's not busy bemoaning the 'big reveal' of his 'secret project.' There's a Mr Turner who's now the cover story's guy bankrolling the thing.`

Reviewer 2: ```suggestion
			`	She looks and blinks at you as if caught slightly off-guard. "Yes, didn't they tell you? In order to keep up the cover they actually have to start making a movie, or at least believable clips from one. The director they found, an eccentric chap called David Cohen, seems to have got quite enthusiastic now he's not busy bemoaning the 'big reveal' of his 'secret project.' There's a Mr Turner who's now the cover story's guy bankrolling the thing.`
```

			choice
				`	"And that is?"`
				`	"What's the deal then?"`

> 			`	Terry crosses her arms and leans back against the bulkhead. "We need a recruitment campaign. We need to attract Hai who lived in shared urban environments with humans enough to be able to fake being humans pretending to be Hai. Which means we need a human touch to this; it can't just come from the Hai's own advertising networks, and we need some star power. Oh, and if you haven't guessed already, you're the one to do it. The Syndicate will offer you a sum of 10,000,000 credits to visit every planet in Hai space and do glossies. I'll obviously stage manage your image, and will have promotional material ready inside of 3 hours."`

Reviewer: Tense confusion was introduced.
```suggestion
			`	Terry crosses her arms and leans back against the bulkhead. "We need a recruitment campaign. We need to attract Hai who've lived in shared urban environments with humans enough to be able to fake being humans pretending to be Hai. Which means we need a human touch to this; it can't just come from the Hai's own advertising networks, and we need some star power. Oh, and if you haven't guessed already, you're the one to do it. The Syndicate will offer you a sum of 10,000,000 credits to visit every planet in Hai space and do glossies. I'll obviously stage manage your image, and will have promotional material ready inside of 3 hours."`
```
